Application/EdFi.Ods.SandboxAdmin/Startup.cs

```diff
@@ -221,10 +218,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
             }
 
             Container = app.ApplicationServices.GetAutofacRoot();
-
-            // Set EF Context
-            DbConfiguration.SetConfiguration(new DatabaseEngineDbConfiguration(Container.Resolve<DatabaseEngine>()));
```

Comment:
What is the alternate mechanism now for the Sandbox Admin tool to setup the correct database engine configuration?